src/main/java/com/blockscore/models/Address.java

```diff
@@ -183,4 +191,6 @@ public String getPostalCode() {
   public String getCountryCode() {
     return countryCode;
   }
+
 }
+
```

Comment:
Hey @chaitanya11, sorry for the delay and thanks for the reminder ping. I should be able to review/audit this later today or tomorrow.